Ravel-Creators/Assets/RavelCreator/Scripts/Editor/CreatorTools/RavelToolbar.cs

```diff
@@ -90,12 +90,12 @@ public class RavelToolbar
         
         //preview environment in ravel web
         if (GUILayout.Button(new GUIContent("Preview", "Build and preview asset bundle"), RavelEditorStying.txtBtnSmall)) {
```

Comment:
RefreshConfig(), We could make the sceneconfiguration a singleton so we don't need to call FinndobjectsofType. Plus There is only one sceneconfiguration anyway per scene